packages/schematics/angular/workspace/files/tslint.json

```diff
@@ -53,56 +44,21 @@
       true,
       "ignore-params"
     ],
-    "no-namespace": false,
     "no-non-null-assertion": true,
     "no-redundant-jsdoc": true,
-    "no-reference": false,
-    "no-reference-import": false,
-    "no-string-literal": false,
     "no-switch-case-fall-through": true,
-    "no-unsafe-finally": false,
     "no-use-before-declare": true,
-    "no-var-requires": false,
```

Comment:
Using require in this form is quite common.  I’m not sure if this is a good idea at this point in time.

Reply:
@clydin I don't mind removing it, but my understanding of the rule was that it doesn't forbid to use require, but only allows it in imports. And as it pushes to use the recommended ES2015 import syntax, I thought it was a good to include it (and I don't recall having to use `var lib = require('lib')` in an CLI project, but maybe you have an example in mind?).

Reply:
I agree with the sentiment but it may be more problematic than useful.
Unfortunately it is also a TypeScript specific extension to the import syntax.  The [standard](https://www.ecma-international.org/ecma-262/9.0/#sec-imports) does not include it.  Users will also receive type errors if the module does not have type definitions available.

Reply:
All right I removed the change.